CONTRIBUTING.md

```diff
@@ -45,9 +40,14 @@ please point me in the right direction (e.g. line numbers) to fix it.
 
 ### Building and Testing Project
 
-Building the project requires [Node.js](https://nodejs.org/) to be installed on your local machine
+Building the project requires [Node.js](https://nodejs.org/) to be installed on your local machine.  Node v14 LTS is recommended.
 
-First, dev dependencies must be installed.  Run `npm install` in both this folder, and in the `./server` folder.
+First, dev dependencies must be installed.  Run `npm install` in both this folder, and in the `./server` folder.  For example:
+
+`npm install`
+`cd server`
+`npm install`
+`cd ..`
```

Comment:
Markdown is picky about this - the above results in all 4 commands on one line. You can either indent each line and not use the tick marks, or just do this, which is to put the commands on separate lines and then 3 tick marks on the line before and after those lines:

```
npm install
cd server
npm install
cd ..
```

Reply:
Done